package.json

```diff
@@ -36,7 +36,6 @@
     "@babel/preset-flow": "^7.10.4",
     "@babel/preset-react": "^7.10.4",
     "@babel/traverse": "^7.11.0",
-    "abort-controller": "^3.0.0",
```

Comment:
Added in https://github.com/facebook/react/pull/21027 but it doesn't look like it was even used in that PR. Maybe something used during development?

Reply:
@eps1lon  “Adds a lightweight fallback shim for environments that don't have an AbortController (e.g. tests running in Node < 15).” from https://github.com/facebook/react/pull/24285

Reply:
Hm. Interestingly that fix is only on the client. It didn’t necessarily fix the server. I guess we just don’t run tests in earlier node environments and we don’t really support them anyway.

Reply:
It should be safe to remove this dependency

Reply:
Given that nothing requires this dependency at the moment, it seems like we can remove it at least until we do require it again. Seems like that might be required for some old node versions, but it seems like they'd need to be tested more extensively anyway.

If someone want's to use abort-controller again, it'd easy to add back the dependency. I'll merge this.